canonical/vanilla-framework
# Remove Suru strips

## Done

Removed the suru strips from the index page and other templates

I kept the current accent colour.

## QA

- Open [demo](insert-demo-url)
- check 
- See that the suru strips are gone and that the alternative looks good

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots
![image](https://user-images.githubusercontent.com/11927929/235153475-de8a093a-94f7-4ccc-8716-62c95159b522.png)

![image](https://user-images.githubusercontent.com/11927929/235153732-918b1a6d-32c6-452e-87ba-7a8125998d17.png)
